lib/core/viewmodels/schedule_viewmodel.dart

```diff
@@ -162,4 +167,19 @@ class ScheduleViewModel extends FutureViewModel<List<CourseActivity>> {
       onError(error);
     }
   }
+
+  Future<void> startDiscovery(BuildContext context) async {
+    if (await _settingsManager.getString(PreferencesFlag.discoverySchedule) ==
+        null) {
+      final List<String> ids =
+          findDiscoveriesByGroupName(context, DiscoveryGroupIds.pageSchedule)
+              .map((e) => e.featureId)
+              .toList();
+
+      Future.delayed(const Duration(milliseconds: 500),
```

Comment:
Should we make this 1 seconds like on More page? I think it should be nice for the user to see the page fully loaded a bit before launching the discovery? @AlexandreMarkus @apomalyn @camillebrulotte your thoughts?

Reply:
Sure ! Mais dans more page c'est la même affaire :D